source/fundamentals/data-formats/documents.txt

```diff
@@ -284,6 +289,96 @@ For more information on retrieving and manipulating MongoDB data, see our
 For more information on this class, see the
 :java-docs:`BsonDocument API documentation </apidocs/bson/org/bson/BsonDocument.html>`.
 
+JsonObject
+-------------
+
+The ``JsonObject`` class acts as a wrapper for JSON strings.
+If you want to work primarily with JSON strings in your application, 
+use the ``JsonObject`` class. For applications that work with JSON
+strings, ``JsonObject`` is a more performant alternative to both ``Document``
+and ``BsonDocument`` as ``JsonObject`` does not create an intermediate
+``Map`` object.
+
+By default, ``JsonObject`` stores :manual:`Extended JSON </reference/mongodb-extended-json/>`.
+You can customize the format of JSON in ``JsonObject`` by specyfying a
+:java-docs:`JsonObjectCodec</apidocs/bson/org/bson/codecs/JsonObjectCodec.html>`
+and passing it a
+:java-docs:`JsonWriterSettings</apidocs/bson/org/bson/json/JsonWriterSettings.html>`
+object. For more information on JSON formats, see
+our :doc:`Extended JSON guide </fundamentals/data-formats/document-data-format-extended-json/>`.
+For more information on specifying codecs, see our 
+:doc:`Codecs guide </fundamentals/data-formats/codecs>`.
+
+In the following code snippet, we show how to instantiate a sample ``JsonObject`` 
```

Comment:
```suggestion
The following snippet shows how to instantiate a sample ``JsonObject`` 
```

Reply:
Thanks for the note @terakilobyte. This language was inherited from other Documents subsections. For example:

> In the following code snippet, we show how to instantiate and build a sample Document instance representing a document containing several different field types:

I think the existing language is clearer, so I don't mind keeping it. If you feel strongly about this revision however, I can change the language in this and the other Documents subsections to match. Let me know your thoughts.

Reply:
Either seems fine to me and I had an extended conversation about usage of "we"/"you" during my first large writing assignments. I prefer what you have as I word using "we" to involve reader and writer, but I think somewhere buried in the style guide is mention that writers should avoid "we".